migrations/versions/2023-09-20T09:32:21Z_add_externalid_to_org.py

```diff
@@ -0,0 +1,31 @@
+"""add externalId to org
+
+Revision ID: aba43062e638
+Revises: 9bd6d9dc6957
+Create Date: 2023-09-20 09:32:21.019505+00:00
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from fastapi_users_db_sqlalchemy.generics import GUID
+
+
+# revision identifiers, used by Alembic.
+revision: str = "aba43062e638"
+down_revision: Union[str, None] = "9bd6d9dc6957"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("organization", sa.Column("external_id", GUID, nullable=False, server_default=sa.text("uuid()")))
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("organization", "external_id")
```

Comment:
would we ever drop columns when going back? Shouldn't the create statement be more clever and ignore existing columns?

Reply:
why won't we? if we want to revert the db to the previous state, we don't want to keep the additional data.